shadow.c

```diff
@@ -555,7 +555,11 @@ static char *get_full_path(const char *filename TSRMLS_DC)
     	if(new_state.cwd) free(new_state.cwd);
         return NULL;
     }
-	return estrndup(new_state.cwd, new_state.cwd_length);
+    char *full_path = estrndup(new_state.cwd, new_state.cwd_length);
```

Comment:
I think indentation here is off by a bit due to preceding closing bracket having the wrong indentation.